numpy/add_newdocs.py

```diff
@@ -1578,7 +1578,9 @@ def luf(lamdaexpr, *args, **kwargs):
 
     Return elements, either from `x` or `y`, depending on `condition`.
 
-    If only `condition` is given, return ``condition.nonzero()``.
+    The rest of this documentation covers only the case where all three
+    arguments are provided. When only `condition` is provided, this function
```

Comment:
This was already true anyway - the `parameters` section did not reflect the `nonzero` case, so it was just odd that the return value did.